CHANGELOG.md

```diff
@@ -5,6 +5,57 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v1.1.0] - 2018-08-21
+
+This version contains 32 contributions from 5 contributors. There are 89 files changed, 2,752 insertions, and 1,013 deletions.
```

Comment:
What do you think about using semicolons instead of commas as the separator here (since there are commas in the numbers)?
```
There are 89 files changed; 2,752 insertions; and 1,013 deletions.
```

Reply:
I'm -1 on semicolons due to incorrect grammar. Honestly I wouldn't mind doing away with the diffstat metrics on the releases. Do they add any value?

Reply:
incorrect grammar? says whom?

Reply:
I had to look this up. I guess you're right -- that semicolons can be used to connect a series of items which themselves contain commas.